src/payment/strategies/moneris/moneris-payment-strategy.spec.ts

```diff
@@ -168,6 +168,15 @@ describe('MonerisPaymentStrategy', () => {
             expect(iframe.src).toMatch(/esqa.moneris.com/);
         });
 
+        it('initialize moneris iframe and sets initailization data labels', async () => {
+            paymentMethodMock.config.testMode = true;
+            await expect(strategy.initialize(initializeOptions)).resolves.toEqual(store.getState());
+
+            const iframe = document.getElementById(iframeId) as HTMLIFrameElement;
+            expect(iframe).toBeTruthy();
+            expect(iframe.src).toMatch(/&display_labels=1&pan_label=Credit%20Card%20Number&exp_label=Expiration&cvd_label=CVV/);
```

Comment:
This doesn't really uses a regex. If we want to do partial matching we could use `expect.stringContaining(string)`: https://jestjs.io/docs/expect#expectstringcontainingstring

```ts
expect(iframe).toHaveAttribute('src', expect.stringContaining('&display_labels=1&pan_label=Credit%20Card%20Number&exp_label=Expiration&cvd_label=CVV'));
```

I noticed we use to match in other tests, I feel like using an exact string match is better in this case.

Reply:
I changed tu use `.tocontain()`
https://jestjs.io/docs/expect#tocontainitem

Reply:
oh! I missed this, I thought it was only for arrays.
> `.toContain` can also check whether a string is a substring of another string.